### ISSUE TITLE
Implement URL parameter to specify remote input file

### ISSUE THREAD

**Issue by lehkost** [gephi/gephi-lite#175]
First off, thanks for this great addition to the Gephi universe. We're using Gephi Lite it in teaching from day 1.

One feature request we would like to bring forward is to allow a GEXF/GraphML source file submitted through a URL parameter. Something like ```&input=https://dracor.org/api/v1/corpora/fre/plays/moliere-avare/networkdata/gexf```.

I know, you can already "Open remote graph" from the Gephi Lite frontend, but using an URL parameter would allow us to link directly from our platform (e.g. https://dracor.org/id/fre000996#downloads) to Gephi Lite with the source file in question.

**Comment by paulgirard** [comment]
It's actually already possible with the `gexf=` parameter: [https://gephi.org/gephi-lite/?gexf=https%3A%2F%2Fraw.githubusercontent.com%2Fgephi%2Fgephi-lite%2Frefs%2Fheads%2Fmain%2Fpackages%2Fgephi-lite%2Fpublic%2Fsamples%2FLes+Miserables.gexf](https://gephi.org/gephi-lite/?gexf=https%3A%2F%2Fraw.githubusercontent.com%2Fgephi%2Fgephi-lite%2Frefs%2Fheads%2Fmain%2Fpackages%2Fgephi-lite%2Fpublic%2Fsamples%2FLes+Miserables.gexf)

Note that the GEXF must be served with CORS header allowing gephi.org domain to load the file if you want to use the publi gephi lite instance. Or you can host Gephi Lite yourself on the same domain thant the one where your GEXF file is served.

**Comment by ingoboerner** [comment]
I the case of DraCor it does't work yet: https://gephi.org/gephi-lite/?file=https%3A%2F%2Fdracor.org%2Fapi%2Fv1%2Fcorpora%2Fger%2Fplays%2Flessing-emilia-galotti%2Frelations%2Fgexf I also got a message that the parameter `gexf` has been renamed to `file`. We will look at the headers, thank you!

**Comment by paulgirard** [comment]
Yes my mistake it should be `file=` and yes the webserver serving your file does not include any CORS header yet so it will not work because of web browser security.
